assets_helper.py

```diff
@@ -28,7 +30,7 @@ def is_active(asset, at_time=None):
 
     if asset['is_enabled'] and asset['start_date'] and asset['end_date']:
         at = at_time or get_time()
-        return asset['start_date'] <= at <= asset['end_date']
+        return asset['start_date'] < at < asset['end_date']
```

Comment:
Aren't date ranges normally inclusive for the beginning? E.g. if you want to play something between 12:00 and 13:00, it would start at 12:00 but it may end 12:59:59.999.